runtime/web_worker.rs

```diff
@@ -56,29 +57,27 @@ impl WorkerId {
   }
 }
 
-type WorkerMessage = ZeroCopyBuf;
-
 /// Events that are sent to host from child
 /// worker.
-pub enum WorkerEvent {
-  Message(WorkerMessage),
+pub enum WorkerControlEvent {
   Error(AnyError),
   TerminalError(AnyError),
   Close,
 }
 
 // Channels used for communication with worker's parent
 #[derive(Clone)]
-pub struct WebWorkerInternalHandle {
-  sender: mpsc::Sender<WorkerEvent>,
-  receiver: Rc<RefCell<mpsc::Receiver<WorkerMessage>>>,
+pub struct WebWorkerSelfHandle {
```

Comment:
I think the previous name was more descriptive

Reply:
Changed it back.